### PR TITLE
Update channel-fuel-testnet.toml for sway 0.61.2

### PR DESCRIPTION
**Pull request by sdankel**
```
cargo run -- channel-fuel-testnet.toml 2024-07-03 forc=0.61.2 forc-wallet=0.8.1 fuel-core=0.28.0 fuel-core-keygen=0.28.0
```

### CODE REVIEW

**Review by JoshuaBatty — APPROVED** [review]

**Review by JoshuaBatty — CHANGES_REQUESTED** [review]
Since we are removing forc-explorer in #644 can we remove it from this toml as well?

**Comment by sdankel** [comment]
Putting this in draft mode until we have release: https://github.com/FuelLabs/sway/pull/6213

File: channel-fuel-testnet.toml [hunk 1]
```diff
@@ -39,58 +39,58 @@ url = "https://github.com/FuelLabs/forc-explorer/releases/download/v0.28.1/forc-
...
 
 [pkg.forc-wallet.target.x86_64-apple-darwin]
-url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.7.1/forc-wallet-0.7.1-x86_64-apple-darwin.tar.gz"
-hash = "001155c56d78e4cebf381ee3a250dfbf62c73b857da0f385abc74a21bb0ea8a6"
+url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.8.1/forc-wallet-0.8.1-x86_64-apple-darwin.tar.gz"
+hash = "5f6e0de39fe684bf08fb666a18322a5927de823efed02b36707b7368c67626c2"
 
 [pkg.fuel-core]
-version = "0.26.0"
+version = "0.28.0"
```
Comment (arboleya) [inline channel-fuel-testnet.toml:61]: Shouldn't this point to `fuel-core` @ `0.30.0`?

testnmet

Comment (sdankel) [reply channel-fuel-testnet.toml:61]: The sway repo is still on 0.28.0

**Review by JoshuaBatty — APPROVED** [review]

**Review by kayagokalp — APPROVED** [review]

**Pull request merged**